### PR TITLE
Deck sorting

### PR DESCRIPTION
**Pull request by alexmiddeleer**

### CODE REVIEW

File: next/components/deck-search-form.js [hunk 1]
```diff
@@ -105,16 +119,30 @@ export default function DeckSearchForm(props) {
           margin: 10px 0;
           width: 100%;
         }
+        :global(.deck-search-form-sort) {
```
Comment (lily-pytel) [inline next/components/deck-search-form.js]: Any reason not to have this in the component itself?

File: postgres/migrations/migration-1.4.1.sql [hunk 2]
```diff
@@ -24,7 +24,7 @@ CREATE OR REPLACE VIEW mythgard.deck_preview as
          deck.name as deck_name,
          deck.created as deck_created,
          array_agg(DISTINCT faction.name) as factions,
-         deck_essence_cost(deck.id)::int as essence_cost,
+         mythgard.deck_essence_cost(deck.id)::int as essence_cost,
```
Comment (lily-pytel) [inline postgres/migrations/migration-1.4.1.sql]: This migration is useless really as it still has the votes bug. I would delete it and just use 1.4.2

File: next/components/deck-search-form.js [hunk 3]
```diff
@@ -127,13 +155,17 @@ export default function DeckSearchForm(props) {
...
             flex-direction: column;
             margin-bottom: 20px;
           }
 
           .action-buttons input {
             margin-bottom: 15px;
+            margin-right: 0;
+          }
+
+          :global(.deck-search-form-sort) {
```
Comment (lily-pytel) [inline next/components/deck-search-form.js]: Any reason not to have this in the component?

Comment (lily-pytel) [reply next/components/deck-search-form.js]: Never mind, I see why!

File: next/components/deck-search-form.js (hunk 1 above)
Comment (lily-pytel) [reply next/components/deck-search-form.js]: I saw why, nvm!

Comment (lily-pytel) [reply next/components/deck-search-form.js]: Actually never mind the never mind - looks like this global is only used here, so why not have it in the component?

File: postgres/migrations/migration-1.4.1.sql (hunk 2 above)
Comment (alexmiddeleer) [reply postgres/migrations/migration-1.4.1.sql]: Oops!

**Comment by alexmiddeleer** [comment]
> This is awesome! My only request is that if we sort by rating, that we show it somewhere - even somewhere temporary.

It's very tempting but I'll let the team voice interest first I think.

**Pull request merged**